be/src/connector/hive_connector.h

```diff
@@ -90,8 +90,8 @@ class HiveDataSource final : public DataSource {
     ObjectPool _pool;
     RuntimeState* _runtime_state = nullptr;
     HdfsScanner* _scanner = nullptr;
-    bool _use_block_cache = false;
-    bool _enable_populate_block_cache = false;
+    bool _use_datacache = false;
```

Comment:
_enable_datacache better?

Reply:
> _enable_datacache better?

It combines the configuration switch in BE and datacache session variable in FE. As an interval variable, I think it is ok.